### PR TITLE
Added example test

### PR DESCRIPTION
**Pull request by kkrampa**
@emord 
http://manage.dimagi.com/default.asp?263218
Please don't merge this.
This is example of test of angular controllers in ICDS Dashboard.

### CODE REVIEW

**Comment by kkrampa** [comment]
@sheelio

File: custom/icds_reports/static/icds_reports/js/spec/adolescent-girls.directive.spec.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        $provide.constant("userLocationId", null);
+    }));
+
+    beforeEach(inject(function($rootScope, $compile, _$httpBackend_, _$location_) {
+        $scope = $rootScope.$new();
+        $httpBackend = _$httpBackend_;
+        $location = _$location_;
+        $httpBackend.expectGET('template').respond(200, '<div></div>');
+        $httpBackend.expectGET('adolescent_girls').respond(200, {
+            report_data: [],
```
Comment (emord) [inline custom/icds_reports/static/icds_reports/js/spec/adolescent-girls.directive.spec.js]: It looks like this is causing this to always return an empty set. Will there be more tests that expect real looking data being returned?

**Review by emord — COMMENTED** [review]
Looks like a good start for this

**Comment by pstudzinski** [comment]
@lwyszomi 
@emord 
Update. 
Please merge if you agree with changes.

https://manage.dimagi.com/default.asp?263218

**Review by emord — APPROVED** [review]

**Pull request merged**